.vscode/launch.json

```diff
@@ -1,5 +1,24 @@
 {
   "version": "0.2.0",
+  "compounds": [
+    {
+      "name": "Both",
+      "stopAll": true,
+      "configurations": ["Launch Extension", "Attach to Server"]
```

Comment:
```suggestion
      "configurations": ["Launch Extension", "Attach to Server"],
```

Reply:
Keep in mind that this is invalid JSON. JSON is not python, you cannot have commas after last element. If I accept this the linter will undo it.